apiServer/taas/utils/torpedoutils.go

```diff
@@ -188,13 +188,19 @@ func ScheduleAppsAndValidate(c *gin.Context) {
 			errStrings = append(errStrings, err.Error())
 		}
 	}
+	namespacesList := make([]string, 0)
+	for _, ctx := range context {
+		namespace := tests.GetAppNamespace(ctx, c.Param("namespacePrefix"))
+		namespacesList = append(namespacesList, namespace)
+	}
 	if len(errStrings) > 0 {
 		c.JSON(http.StatusInternalServerError, gin.H{
 			"error": errStrings,
 		})
 	} else {
 		c.JSON(http.StatusOK, gin.H{
-			"message": "Apps Created and Validated successfully",
+			"message":   "Apps Created and Validated successfully",
```

Comment:
pls change to "App is created and validated successfully"

Reply:
done